src/components/layout/nav.js

```diff
@@ -108,15 +83,33 @@ const NavRightContainer = styled.div`
 class Nav extends React.Component {
     state = {
         enable_move: false,
+        show_modal: false,
+        active_tab: '',
+    }
+
+    toggleModal = e => {
```

Comment:
you can use the modal hook instead of this (check about us/index/keep safe for example)
+ change the handlescroll to a hook as well